webhook/webhook.go

```diff
@@ -0,0 +1,11 @@
+package webhook
+
+import (
+	"net/http"
+
+	"github.com/micromdm/micromdm/pubsub"
+)
+
+type ListeningWebhook interface {
```

Comment:
remove this interface and used an exported struct first. We can define the appropriate interface later when it becomes required.